src/classes/TDTM_Global_API.cls

```diff
@@ -194,7 +201,7 @@ global class TDTM_Global_API {
             this.className = className;
             this.targetObject = targetObject;
             this.actions = actions;
-            this.loadOrder = loadOrder;
```

Comment:
Don't we still need to populate the old property (loadOrder) for backwards compatibility? (In case the was any code outside HEDA retrieving loadOrder).